jsoc/gsoc/javis.md

```diff
@@ -1,69 +1,51 @@
+**Recommended Skills:** Familiarity with profiling, caching approaches, and performance testing
 
-## Linear algebra
+**Difficulty:** Medium
 
-**Mentors**: Ole Kröger, Jacob Zelko
+As Javis's interface is largely stabilized and Javis is finding use in different applications, it is now time to deal with one of Javis's greatest pain points: slowness and high memory usage for large animations.
+While creating an animation in Javis, there is much room for performance improvements such as in the area of creating Objects and Actions, managing the data structures for Objects and Actions, rendering an animation, and handling different media formats (such as gif and mp4).
+For this specific project, a student will work with Ole and Arsh to create a profiling scheme for Javis to identify performance bottlenecks and measure allocations, determine caching and memory flexible modes of rendering animations with tools such as [FFMPEG.jl](https://github.com/JuliaIO/FFMPEG.jl), and finish implementing live streaming of animations.
+The goal for this project will not be to fully fix all identified performance issues but rather to identify and catalogue them for further development by Javis maintainers and contributors.
 
-**Recommended skills**: Basic to intermediate knowledge about linear algebra.
+## Building Novel Animation Abilities for Javis
 
-**Difficulty:** Easy 
+**Mentors:** Jacob Zelko, Giovanni Puccetti
 
-**Description**: Linear algebra is of invaluable importance all across different fields of mathematics and engineering.
-Enabling the easy creation of visualizations regarding rotations, matrices and other concepts is helpful in educating students about this amazing branch mathematics.
-Here are a few issues related to tasks that could be worked on to bring about this capability:
+**Recommended Skills:** General understanding of Luxor and the underlying structure of Javis
 
-- Vectors are foundational to linear algebra, help Javis visualize them! [Issue #31](https://github.com/Wikunia/Javis.jl/issues/31)
+**Difficulty:** Medium
 
-- Drawing backgrounds such as grids can assist in easy viewing of complicated mathematical operations such as rotations. [Issue #38](https://github.com/Wikunia/Javis.jl/issues/38)
+Javis's interface has matured to a great point - but we believe Javis can do even more!
+Although Javis can do complex transformations such as morphing one polygon to another, Javis is capable of more than that.
+In this project, a student will work with Jacob and Giovanni to create new animation abilities for Javis to handle different coordinate systems, developing new types of shorthand expressions for object creation known as JObjects, further developing morphing, building out the flexibility of layers, and more.
+A student is encouraged to come to this project with new ideas for what animations Javis can do and to reach out to Jacob and Giovanni to begin discussions early.
```

Comment:
Hi @TheCedarPrince looks very good, one thing that might be added as an example is a link to this [PR](https://github.com/JuliaAnimators/Javis.jl/pull/434) for a idea of what a new interface can look like. It is not complete but almost is and I think it fits in the type of things that would be good to do in this project. Other than that I think it is good!